acceptance/tests/reports/failover_master.rb

```diff
@@ -0,0 +1,56 @@
+test_name "The report specifies which master was contacted during failover" do
+  master_reportdir = create_tmpdir_for_user(master, 'report_dir')
+  master_port = 8140
+
+  master_opts = {
```

Comment:
And this is what happens when you try to take examples from multiple tests without fully understanding how they work xD